encrypted-media/content/drmconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "com.widevine.alpha": {
+        "drmtoday": true,
+        "serverURL": "https://lic.staging.drmtoday.com/license-proxy-widevine/cenc/",
+        "httpRequestHeaders": {
+            "dt-custom-data": "eyJ1c2VySWQiOiIxMjM0NSIsInNlc3Npb25JZCI6ImV3b2dJQ0p3Y205bWFXeGxJaUE2SUhzS0lDQWdJQ0p3ZFhKamFHRnpaU0lnT2lCN0lIMEtJQ0I5TEFvZ0lDSnZkWFJ3ZFhSUWNtOTBaV04wYVc5dUlpQTZJSHNLSUNBZ0lDSmthV2RwZEdGc0lpQTZJR1poYkhObExBb2dJQ0FnSW1GdVlXeHZaM1ZsSWlBNklHWmhiSE5sTEFvZ0lDQWdJbVZ1Wm05eVkyVWlJRG9nWm1Gc2MyVUtJQ0I5TEFvZ0lDSnpkRzl5WlV4cFkyVnVjMlVpSURvZ1ptRnNjMlVLZlFvSyIsIm1lcmNoYW50IjoiY2FibGVsYWJzIn0K"
```

Comment:
@greg80303 Do you mean that there is a component that is doing a byte-for-byte comparison of the base64 or JSON encoding ?

I was just thinking that in this file we would undo both the base64's (& JSON) for transparency and then before sending the object we would do the JSON and base64 encoding. What we could end up with would be different from the above because JSON encoding is not canonical.